tests-trigger

```diff
@@ -75,7 +76,7 @@ def trigger_pull(api, opts):
         changes = {"state": "pending", "description": github.NOT_TESTED_DIRECT, "context": context}
 
         # Keep the old link for reference, until testing starts again
-        link = status.get("target_url", None)
+        link = get_str(status, "target_url", None)
```

Comment:
mental note: we're allowed to have walruses now `:=`

Reply:
Thanks for pointing out! done in #6278